build/linux/installer/conf/fluent-bit-network-flow-logs.conf

```diff
@@ -0,0 +1,31 @@
+[INPUT]
+    Name                tail
+    Alias               oms_networkflow_input
+    Tag                 oms.networkflow.*
+    Path                /var/log/acns/hubble/events.log
+    Read_from_Head      true
+    DB                  /var/opt/microsoft/docker-cimprov/state/network_flow.db
+    DB.Sync             Off
+    Parser              json
+    Rotate_Wait         20
+    Refresh_Interval    5
+    storage.type        filesystem
+    Skip_Long_Lines     On
+    Ignore_Older        2m # configurable
+
+[FILTER]
+    Name                throttle
+    Alias               oms_networkflow_throttle
+    Match               oms.networkflow.*
+    Rate                1200
```

Comment:
default looks good but should be configurable in case if any customer to configure different values or we need to handle incident.